examples/data-objects/webflow/src/test/layout.spec.ts

```diff
@@ -42,7 +42,10 @@ describeCompat("Layout", "LoaderCompat", (getTestObjectProvider) => {
 	let layout: Layout;
 
 	let provider: ITestObjectProvider;
-	before(async () => {
+	before(async function () {
+		// Initialization of 'compat LTS ^1.3.4 - old loader' compat variant can take a couple seconds.
+		this.timeout(10000);
+
```

Comment:
I observed this using a codespaces VM, and it looks like we're right about at default 2s timeout.

Reply:
Interesting. Wonder what it is about our two PRs - it feels like too much coincidence, but maybe that is all it is.